komodo/show_version.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+import argparse
+import yaml
+import os
+import re
+from pathlib import Path
+import configparser
+from typing import Union
+from typing import Optional
+
+
+def guess_release() -> str:
+    """
+    Try to sense the Komodo release from the active environment.
+    """
+    try:
+        return os.environ["KOMODO_RELEASE"]
+    except KeyError:
+        raise Exception("Could not sense the name of the release, please provide it.")
+
+
+def read_config(fname: Union[str, Path]) -> dict:
+    """
+    Read an INI file (aka config file) that does not have sections.
+    Creates a temporary 'MAIN' section and pass bask its contents.
+    """
+    config = configparser.ConfigParser()
+    with open(fname, "rt") as f:
+        config.read_string("[MAIN]\n" + f.read())
+    return dict(config["MAIN"])
+
+
+def guess_komodoenv_path(release: str) -> Path:
+    """
+    Use the release name to find the 'real' release path, but from a
+    komodoenv environment. These environments overwrite the environment
+    variable $KOMODO_RELEASE and $PATH, but they store the original path
+    in the komodoenv at 'root/pyvenv.cfg'.
+
+    Args:
+        release (str): The name of the release, e.g. from $KOMODO_RELEASE.
+
+    Returns:
+        str: The path to the release, where the 'root/bin' folder is.
+    """
+    config = read_config(Path(release) / "root" / "pyvenv.cfg")
+    path, *_ = config["home"].split("/root/bin")
```

Comment:
(`maxsplit=1` is an alternative to `*_`)

Reply:
Not sure, still requires at least the comma (i.e. `path, = str.split(maxsplit=1)`) because the split always returns at least two items. Will keep current unless you really don't like that syntax.

Reply:
ah, then keep it :)